db/data.py

```diff
@@ -200,7 +218,8 @@ def _insert_lowlevel_json(connection, ll_id, data_json, data_sha256, version_id)
             return
 
         try:
-            ll_id = _insert_lowlevel(connection, mbid, build_sha1, is_lossless_submit, is_mbid)
+            submission_offset = _get_submission_offset(connection, mbid)
+            ll_id = _insert_lowlevel(connection, mbid, build_sha1, is_lossless_submit, is_mbid, submission_offset)
```

Comment:
We should have a test here to ensure that the offset is the expected value (0 if we add an mbid once, 1 if we add it twice, and another mbid will be 0 again)